README.md

```diff
@@ -106,6 +107,17 @@ by `CMake` in some cases (see below for further details).<br/>
 Because of that, users don't have to install it to run the tests or when `uvw`
 libraries are compiled through `CMake`.
 
+## Meson
+
+You can use `uvw` with [meson](https://mesonbuild.com/) by simply adding it to your `subprojects` directory in your project. Currently, `uvw` is not available with `meson wrap`, but this will change with the release of `uvw` 3.2.1.
```

Comment:
Is it version 3.3.x?

Reply:
It would be whatever the next release of uvw is, I assumed this would be 3.2.1, but if there were breaking changes then yeah, 3.3.x.